x-pack/plugins/infra/public/pages/metrics/metrics_explorer/hooks/use_metrics_explorer_data.ts

```diff
@@ -67,13 +67,7 @@ export function useMetricsExplorerData(
           body: JSON.stringify({
             forceInterval: options.forceInterval,
             dropLastBucket: options.dropLastBucket != null ? options.dropLastBucket : true,
-            metrics:
-              options.aggregation === 'count'
-                ? [{ aggregation: 'count' }]
-                : options.metrics.map((metric) => ({
```

Comment:
Why don't we need a map here anymore?

Reply:
The map was just mapping it back to the same format. I'm not sure why it was there in the first place.